modules/fedimint-gw-client/src/lib.rs

```diff
@@ -0,0 +1 @@
+// TODO: Implement GW client
```

Comment:
You need to add `(pub) mod …` entries for the files with the diagrams so they are rendered